Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -662,6 +662,8 @@ public function testMessengerIntegration()
         $extension->load([$config], $container);
 
         $this->assertNotNull($middlewarePrototype = $container->getDefinition('messenger.middleware.doctrine_transaction'));
+        $this->assertNotNull($middlewarePrototype = $container->getDefinition('messenger.middleware.doctrine_ping_connection'));
+        $this->assertNotNull($middlewarePrototype = $container->getDefinition('messenger.middleware.doctrine_close_connection'));
```

Comment:
This changes the test - the assertCount call below needs to be duplicated as well.

Reply:
added